package.json

```diff
@@ -107,5 +109,18 @@
     "socketcluster": "4.1.0",
     "socketcluster-client": "4.1.2",
     "url": "0.11.0"
+  },
+  "xo": {
+    "esnext": true,
+    "space": 2,
+    "env": [
+      "browser",
+      "node",
+      "webpack"
+    ],
+    "extends": "xo-react",
+    "globals": [
+      "__PRODUCTION__"
+    ]
   }
 }
```

Comment:
XO is a wrapper around eslint, making it easier to configure and providing linter interfaces that can call `--fix` from the editor. Love atom-linter-xo…

I don't see this as polluting package.json, it accurately describes the project coding style in a file that is always there. 

Anyway, if you don't like that there's the manual way as described at https://github.com/sindresorhus/xo#why-not-eslint